File: agent/bindings/kotlin/lib/src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated/Webview_CreateWebviewPanelParams.kt
@file:Suppress("FunctionName", "ClassName", "unused", "EnumEntryName", "UnusedImport")
package com.sourcegraph.cody.agent.protocol_generated;

data class Webview_CreateWebviewPanelParams(
  val handle: String,
  val viewType: String,
  val title: String,
  val showOptions: ShowOptionsParams,
  val options: WebviewCreateWebviewPanelOptions,
)

File: agent/bindings/csharp/WebviewCreateWebviewPanelParams.cs
using System.Text.Json.Serialization;

namespace Cody.Core.Agent.Protocol
{
  public class WebviewCreateWebviewPanelParams
  {
    [JsonProperty(PropertyName = "handle")]
    public string Handle { get; set; }
    [JsonProperty(PropertyName = "viewType")]
    public string ViewType { get; set; }
    [JsonProperty(PropertyName = "title")]
    public string Title { get; set; }
    [JsonProperty(PropertyName = "showOptions")]
    public ShowOptionsParams ShowOptions { get; set; }
    [JsonProperty(PropertyName = "options")]
    public WebviewCreateWebviewPanelOptions Options { get; set; }
  }
}
